src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
+		return;
+	}
+	var subCmd = args[0];
+	var client = this;
+	var irc = network.irc;
+	var otrStore = client.otrStore;
+	var session = otrStore.getSession(chan.title, network);
+
+	if (subCmd === "init") {
```

Comment:
Yep, I'll switch to a `switch` (no pun intended) or other nice syntax when we have more than one command, in a further PR then.